Misc/NEWS.d/next/Core and Builtins/2022-05-06-03-53-47.gh-issue-57684.ITjOhD.rst

```diff
@@ -0,0 +1,4 @@
+Add the :option:`-p` option to ignore the :option:`-P` option and the
+:envvar:`PYTHONSAFEPATH` environment variable. The :option:`-p` option takes
+precedence over the :option:`-I` option (isolated mode). Patch by Victor
+Stinner.
```

Comment:
As before, I'd suggest being clearer here that `-p` doesn't override `-I` entirely, just the `-P` it implies.

```suggestion
:envvar:`PYTHONSAFEPATH` environment variable. The :option:`-p` option also
takes precedence over the :option:`-P` implied by the :option:`-I` option
(isolated mode). Patch by Victor Stinner.
```